pkg/sdn/plugin/node.go

```diff
@@ -204,9 +202,23 @@ func (node *OsdnNode) Start() error {
 		}
 	}
 
-	// Wait for kubelet to init the plugin so we get a knetwork.Host
-	log.V(5).Infof("Waiting for kubelet network plugin initialization")
-	<-node.kubeletInitReady
+	log.V(5).Infof("Waiting for kubelet and network plugin initialization")
+	err = kwait.ExponentialBackoff(
```

Comment:
So... do you want me to change it to use PollInfinite()? Or just bump the timeout to like 10 minutes or something?